quit/web/modules/endpoint.py

```diff
@@ -128,11 +131,14 @@ def sparql(branch_or_ref):
 
     try:
         if queryType == 'SelectQuery':
-            return create_result_response(res, resultSetMimetypes[mimetype])
+            response = create_result_response(res, resultSetMimetypes[mimetype])
         elif queryType == 'AskQuery':
-            return create_result_response(res, askMimetypes[mimetype])
+            response = create_result_response(res, askMimetypes[mimetype])
         elif queryType in ['ConstructQuery', 'DescribeQuery']:
-            return create_result_response(res, rdfMimetypes[mimetype])
+            response = create_result_response(res, rdfMimetypes[mimetype])
+        if commitid:
+            response.headers["X-CurrentCommit"] = commitid
```

Comment:
If an Update Query was successfully executed but has no effect on the store, it might be useful to return the current commit id.